readme.md

```diff
@@ -1,6 +1,6 @@
 # ncc
 
-[![Build Status](https://circleci.com/gh/zeit/ncc.svg?&style=shield)](https://circleci.com/gh/zeit/workflows/ncc)
+[![CI Status](https://badgen.net/github/checks/zeit/ncc?label=CI)](https://github.com/zeit/ncc/actions?workflow=CI)
```

Comment:
Why not use the official one from GH?

```suggestion
[![CI Status](https://github.com/zeit/ncc/workflows/CI/badge.svg)](https://github.com/zeit/ncc/actions?workflow=CI)
```

Reply:
I didn't know there was an official badge 🤯

Reply:
There is. The thing is why use official badges. One of the great things of services like Badgen and Shields is that all the badges are consistent, and definitely looking better.